### DISCUSSION TITLE
Welcome to Mecha CMS Forums!

### DISCUSSION THREAD

**Discussion by taufik-nurrohman** [mecha-cms/mecha#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KompiAjaib** [comment]
Hadir mas, ikut meramaikan forum nih hehe...

**Reply by taufik-nurrohman** [reply]
Silakan pak kalau sudah siap nyoba boleh tanya-tanya. Tab diskusi insya Allah lebih bebas dibandingkan dengan tab isu.

**Reply by igoynawamreh** [reply]
🥇

**Comment by bungomedia** [comment]
Bismillah, ikut merayakan forum ini meskipun terlambat semoga tetap pada semangat berkreasi.

Salam Sehat Selalu dan Semoga Selalu diberikan Keberkahan Dunia dan Akhirat.
Mohon Tutorialnya untuk pemasangan Script ini yang rencana saya mau saya aplikasikan ke [Website nuruljannah](https://nuruljannah.or.id/)

Trims

**Comment by yuliakbard** [comment]
Hadir, sudah lebih 1 tahun jadi pengguna Mecha CMS, ikut tumbuh dan berkembang bersama Mecha CMS

[Loker Sumatera](https://lokersumatera.com)

**Reply by bungomedia** [reply]
Waah cocok, sukses selalu mas, apa kira-kira keluh kesahnya selama menggunakan cms mecha mas, cerita dong

**Reply by yuliakbard** [reply]
Mecha CMS termasuk CMS yang paling ringan konsumsi ruang penyimpanan nya. kalau ada problem tinggal ajukan pertanyaan di Github Discussion Mecha ini. bakal dibantu sama founder nya kok. 

Keluh kesah nya, palingan harus sedikit bersabar dengan update-an mecha nya karena developer aktif cuma kang taufiq aja.

**Reply by taufik-nurrohman** [reply]
Kalau masa-masa merintis gini yang jelas bakal banyak breaking changes di sisi core, karena saya lebih mengutamakan ukuran berkas yang kecil. Kadang mungkin dulu pakai ekstensi ini/itu bisa tapi setelah update jadi tidak bisa. Kuncinya ya kabar-kabar saja ke saya insya Allah nanti saya bantu.
